src/loom-provider.ts

```diff
@@ -383,6 +372,10 @@ export class LoomProvider {
     this._client.disconnect()
   }
 
+  reset() {
```

Comment:
This one line function is still as pointless as when I removed it, why did you add it back?

Reply:
Reset function isn't pointless is used internally by the `Web3` to reset the callbacks if necessary by the request manager `https://github.com/ethereum/web3.js/blob/1.x/packages/web3-core-requestmanager/src/index.js#L227-L228`

Reply:
OK, thanks for the clarification, I've added a comment to ensure it's not removed again in the future.